tavern/testutils/pytesthook/item.py

```diff
@@ -194,6 +194,16 @@ def runtest(self):
                     stage["name"] = stage["id"]
 
             run_test(self.path, self.spec, self.global_cfg)
+
+            # TODO: Will this also run after test failure?
```

Comment:
This won't be run if the test fails, you would need to put it in a 'finally' block or that to work.